From 2e33433913f5e8eff69313d461ba5892aa66563e Mon Sep 17 00:00:00 2001
From: Mario Schuknecht <mario.schuknecht@iris-sensing.com>
Date: Tue, 8 Jul 2025 16:28:57 +0200
Subject: [PATCH] Activate only sdhc 3 and 4 HYP-31820

Activate only sdhc 3 and 4 so that they are listed as mmc0 and mmc1.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Mario Schuknecht <mario.schuknecht@iris-sensing.com>
---
 arch/arm/dts/imx6qdl-nitrogen6x.dtsi | 2 +-
 arch/arm/dts/imx6qdl.dtsi            | 6 ++++--
 2 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/arch/arm/dts/imx6qdl-nitrogen6x.dtsi b/arch/arm/dts/imx6qdl-nitrogen6x.dtsi
index 904d5d051d6..44714bddcb8 100644
--- a/arch/arm/dts/imx6qdl-nitrogen6x.dtsi
+++ b/arch/arm/dts/imx6qdl-nitrogen6x.dtsi
@@ -662,7 +662,7 @@
 	vmmc-supply = <&reg_wlan_vmmc>;
 	cap-power-off-card;
 	keep-power-in-suspend;
-	status = "okay";
+	status = "disabled";
 
 	#address-cells = <1>;
 	#size-cells = <0>;
diff --git a/arch/arm/dts/imx6qdl.dtsi b/arch/arm/dts/imx6qdl.dtsi
index 4f7fefc14d0..458da8b1d1d 100644
--- a/arch/arm/dts/imx6qdl.dtsi
+++ b/arch/arm/dts/imx6qdl.dtsi
@@ -32,10 +32,12 @@
 		i2c1 = &i2c2;
 		i2c2 = &i2c3;
 		ipu0 = &ipu1;
+	/*
 		mmc0 = &usdhc1;
 		mmc1 = &usdhc2;
-		mmc2 = &usdhc3;
-		mmc3 = &usdhc4;
+	*/
+		mmc0 = &usdhc3;
+		mmc1 = &usdhc4;
 		serial0 = &uart1;
 		serial1 = &uart2;
 		serial2 = &uart3;
-- 
2.43.0

